src/users/containers/user-details/user-details.component.ts

```diff
@@ -1,12 +1,16 @@
 import { Component, OnDestroy, OnInit } from '@angular/core';
-import { User } from '@hmcts/rpx-xui-common-lib';
+import { User, UserAccessType } from '@hmcts/rpx-xui-common-lib';
 import { Actions, ofType } from '@ngrx/effects';
 import { select, Store } from '@ngrx/store';
 import { combineLatest, Observable, Subject, Subscription, takeUntil } from 'rxjs';
 
 import * as fromRoot from '../../../app/store';
 import * as fromStore from '../../store';
+import * as fromOrgStore from '../../../organisation/store';
 import { ActivatedRoute } from '@angular/router';
+import { OrganisationAccessType } from 'src/models';
+
+import { userAccessTypesExample } from '../manage-user/temp-data';
```

Comment:
I believe this is unused now and can be removed. please double check all your new imports